src/api/entities/Subsidy/index.ts

```diff
@@ -64,6 +126,31 @@ export class Subsidy extends Entity<UniqueIdentifiers, HumanReadable> {
    */
   public quit: NoArgsProcedureMethod<void>;
 
+  /**
+   * Set allowance for this Subsidy relationship.
+   *
+   * @note Only the subsidizer is allowed to set the allowance
+   *
+   * @throws if the allowance to set is equal to the current allowance
+   */
+  public setAllowance: ProcedureMethod<Pick<ModifyAllowanceParams, 'allowance'>, void>;
+
+  /**
+   * Increase allowance for this Subsidy relationship.
+   *
+   * @note Only the subsidizer is allowed to increase the allowance
+   */
+  public increaseAllowance: ProcedureMethod<Pick<ModifyAllowanceParams, 'allowance'>, void>;
+
+  /**
+   * Decrease allowance for this Subsidy relationship.
+   *
+   * @note Only the subsidizer is allowed to decrease the allowance
+   *
+   * @throws if the allowance to decrease is not less than the existing allowance
```

Comment:
```suggestion
   * @throws if the amount to decrease by is more than the existing allowance
```

(nit) I think "more than" is easier to understand than "not less than".